File: ClientApp/src/guest.ts
export interface Guest {
  GuestId: number;
  name: string;
  email: string;
  friends: number;
  rsvp: string;
  message: string;
  eventId: number;
}

export const GUESTS: Guest[] = [
  {
    GuestId: 1,
    name: "Santa Claus",
    email: "nick@northpole.com",
    friends: 1,
    rsvp: "yes",
    message: "Ho ho ho",
    eventId: 1
  },
  {
    GuestId: 2,
    name: "The Grinch",
    email: "green@seuss.com",
    friends: 0,
    rsvp: "no",
    message: "...",
    eventId: 1
  }
];

File: Database/Entities/Guest.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RSVPTake3.Database.Entities
{
    public class Guest
    {
        [Key]
        public int GuestId { get; set; }
        [Required]
        [Column(TypeName = "varchar(20)")]
        public string Name { get; set; }
        public string Email { get; set; }
        public string RSVP { get; set; }
        public int Friends { get; set; }
        public string Message { get; set; }
        public int EventId { get; set; }

    }
}
